### PR TITLE
Allow running installer on 10.5

### PR DESCRIPTION
**Pull request by MikeMcQuaid**
It requires `--insecure` but if we can make it work: we may as well.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,8 +1,8 @@
 language: ruby
 os: osx
-env: OSX=10.11
-osx_image: xcode7.3
-rvm: system
+env: OSX=10.10
+osx_image: xcode6.4
+rvm: 1.8.6-p287
```
Comment (reitermarkus) [inline .travis.yml]: Why can't this stay at `system`, are you “simulating” 10.5 with this? And is the `OSX` variable actually doing anything?

Comment (MikeMcQuaid) [reply .travis.yml]: > are you “simulating” 10.5 with this

Kinda, yes.



> And is the OSX variable actually doing anything?

It's describing the version in the image for quick reference in the Travis UI.

Comment (reitermarkus) [reply .travis.yml]: It's describing the version in the image for quick reference in the Travis UI.

> Ah, I see.

**Comment by reitermarkus** [comment]
https://github.com/Homebrew/install/pull/108/files#diff-19ad89bc3e3c9d7ef68b89523eff1987L94 should be `.map { |p| p.to_i }` for Ruby 1.8.

**Pull request merged**